src/pybind/mgr/mgr_module.py

```diff
@@ -339,6 +345,17 @@ def CLIWriteCommand(prefix, args="", desc=""):
 def _get_localized_key(prefix, key):
     return '{}/{}'.format(prefix, key)
 
+# common/options.h
+OptionType = Union[
+    int,
+    str,
+    float,
+    bool,
+    ]
+
+
+# common/options.h: value_t
+ValueType = Union[bool, int, float, str]
```

Comment:
the variable name of `ValueType` is way too general in this context. can we use `OptionType` instead? also `OptionType` is not used anywhere in this changeset.

Reply:
I just wanted to adapt the C++ type name (`value_t`) and AFAICR it's not the same as the `OptionType` (which should work for typing `get_module_option` and related... I missed that part), as it's used for certain parameter types (`min`, `max`).

Reply:
please note, `value_t` is defined as a member of `Option`. an alternative is to enclose this type definition in `Option`?

Reply:
Removed and replaced with single `OptionValue` type.